changelogs/current.yaml

```diff
@@ -59,6 +59,8 @@ new_features:
 - area: http
   change: |
     added default-false ``envoy.reloadable_features.http1_use_balsa_parser`` for experimental BalsaParser.
+  change: |
+    added ``envoy_reloadable_features_allow_upstream_filters`` for experimental upstream filters.
```

Comment:
Should this be `envoy.reloadable_features.allow_upstream_filters` instead of `envoy_reloadable_features_allow_upstream_filters`

Reply:
yep.  though this reminds me: @phlax in this instantiation of the PR the guard is true but even when it's false the scripts require comments.  I don't think we should require docs for guards which are false.